src/less/components/_forms-help.less

```diff
@@ -183,32 +186,26 @@
 // Markup:
 // <div class="form-group form-group__inline">
 //     <label class="control-label">Service Name</label>
-//     <div class="form-readonly-text form-readonly-text__inline">read-only content styling with inline level help </div>
-//     <span class="section-help help-inline help-inline--section help-inline--tooltip section-help--tooltip__inline">
+//     <div class="form-readonly-text">read-only content styling with inline level help </div>
+//     <span class="section-help help-inline help-inline--section help-inline--tooltip">
 //         <span class="hyicon hyicon-helpoff help-icon-inline"></span>
 //         <span class="help-block-inline">I'm a additional Inline Help Text - inlined</span>
 //     </span>
 // </div>
 //
-// Component 5.7
+// Component 5.8
 
 .form-group__inline{
     label{
         display: block;
         margin-bottom: 0;
     }
-    .form-readonly-text__inline{
+    .form-readonly-text{
```

Comment:
@xak @joseegm  Good point, I just double checked with @saadmhybris and Core Builder Project and Alligators (Package and Operator view in Builder) and no-one use it and never used. Therefore I'll suggest to deprecate the .form-readonly-text__inline and announce it in release notes. Please let me know what you think?